### ISSUE TITLE
Feauture Suggestion/Request: Custom Tag/Tag Sequence

### ISSUE THREAD

**Issue by diego898** [aziz/PlainTasks#146]
Hello! First, thank you for this great package! I'd just like to reopen a feature request/recommendation made some time ago as I think it would be an invaluable addition to plaintasks: @mbacou made this some time ago in #9 and @vovkkk said to feel free to open a new issue with a specific suggestion. 

Seconding @mbacou I think it would be great to have:
- support for _custom tags_ - allowing you to customize tag word, autocomplete, background and foreground color
- a tag sequence to cycle tags through, for example Using `ctrl+d` on a task would then cycle through the sequence.

```
{[
"task_sequence" : [ "none", "scheduled", "active", "|", "done" , "|", "cancelled"]
// or another example,
"task_sequence" : [ "none", "active", "feedback" , "|", "done", "delegated", "|", "cancelled" ]
]}
```
- support for the `@due` tag, `@priority` tag, etc.

**Comment by darkled** [comment]
Is this sequence already implemented?

**Comment by vovkkk** [comment]
No. I tried but code turns to be too complecated and unreliable, feel free to send pull request (of course if your implementation is reliable).

**Comment by PixelT** [comment]
> support for custom tags - allowing you to customize tag word, autocomplete, background and foreground 

+1
